raddb/sites-available/coa-relay

```diff
@@ -256,14 +238,16 @@ server coa {
 		#  Report what we did
 		#
 		if (&control.Tmp-Integer-0) {
-			update reply {
-				&Reply-Message += "Sent updates for %{control.Tmp-Integer-0} active sessions"
+			&reply.Reply-Message += {
```

Comment:
don't use `+=` .  I had similar comments in previous PRs.  I don't want to repeat similar comments in new PRs